### DISCUSSION TITLE
Error after install in Windows 11 - BrowserTeest.php

### DISCUSSION THREAD

**Discussion by wbetalleluzd** [livewire/livewire#5801] [7. Bugs (archived)]
BrowserTest.php

![image](https://github.com/livewire/livewire/assets/4992150/a22689e6-5aec-4465-908e-1004f6e37365)

**Comment by joshhanley** [comment]
This issue has been converted to a discussion as we are now using discussions to track bugs and ideas.
See this announcement for more details https://github.com/livewire/livewire/discussions/3147

**Comment by joshhanley** [comment]
@wbetalleluzd please don't open issues, please only open discussions. See my comment above.
